app/app/src/main/java/me/bickositieff/raspio/ui/playlist/SongAdapter.kt

```diff
@@ -6,7 +6,7 @@ import androidx.recyclerview.widget.RecyclerView
 import me.bickositieff.raspio.databinding.SongItemBinding
 import me.bickositieff.raspio.ui.models.Song
 
-class SongAdapter(
+open class SongAdapter(
```

Comment:
I don't think this is still needed, is it?
```suggestion
class SongAdapter(
```